ios/clever_ads_solutions.podspec

```diff
@@ -1,23 +1,22 @@
 #
 # To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
-# Run `pod lib lint cas.podspec` to validate before publishing.
+# Run `pod lib lint clever_ads_solutions.podspec` to validate before publishing.
 #
 
 Pod::Spec.new do |s|
   s.name             = 'clever_ads_solutions'
-  s.version          = '0.3.1'
+  s.version          = '0.3.2'
   s.summary          = 'A new flutter plugin project.'
   s.description      = <<-DESC
 A new flutter plugin project.
                        DESC
   s.homepage         = 'http://example.com'
   s.license          = { :file => '../LICENSE' }
   s.author           = { 'CAS.ai' => 'email@example.com' }
-  s.source           = { :path => '.' }
-  s.source_files = 'Classes/**/*'
-  s.source = { :path => 'https://github.com/cleveradssolutions/CAS-Specs.git' }
+  s.source           = { :git => 'https://github.com/cleveradssolutions/CAS-Specs.git' }
+  s.source_files     = 'Classes/**/*'
```

Comment:
For a local Podspec, `source` should simply be `{ :path => '.' }`.  
CAS-Specs.git is the source of module specifications, but not the source of `source_files`.